ceci/stage.py

```diff
@@ -1113,6 +1170,9 @@ def generate_command(cls, inputs, config, outputs, aliases=None):
                 raise ValueError(f"Missing input location {aliased_tag} {str(inputs)}") from msg
             flags.append(f"--{tag}={fpath}")
 
+        if instance_name is not None:
```

Comment:
I think this should only happen when instance_name is not the same as class name?

Reply:
Or instance_name is not the same as cls.name (Not quite the same thing).